web/src/pages/Vulnerability/VulnerabilityPage.jsx

```diff
@@ -28,63 +23,66 @@ export function Vulnerability() {
     isLoading: pteamIsLoading,
   } = useGetPTeamQuery(pteamId, { skip });
   const {
-    data: serviceTag,
-    error: serviceTagError,
-    isLoading: serviceTagIsLoading,
-  } = useGetTagQuery(serviceTagId, { skip });
+    data: vuln,
+    error: vulnError,
+    isLoading: vulnIsLoading,
+  } = useGetVulnQuery(vulnId, { skip });
   const {
-    data: topic,
-    error: topicError,
-    isLoading: topicIsLoading,
-  } = useGetTopicQuery(topicId, { skip });
-  const {
-    data: topicActions,
-    error: topicActionsError,
-    isLoading: topicActionsIsLoading,
-  } = useGetTopicActionsQuery(topicId, { skip });
+    data: vulnActions,
+    error: vulnActionsError,
+    isLoading: vulnActionsIsLoading,
+  } = useGetVulnActionsQuery(vulnId, { skip });
 
   if (skip) return <></>;
   if (pteamError) throw new APIError(errorToString(pteamError), { api: "getPTeam" });
-  if (serviceTagError) throw new APIError(errorToString(serviceTagError), { api: "getTag" });
-  if (topicError) throw new APIError(errorToString(topicError), { api: "getTopic" });
-  if (topicActionsError)
-    throw new APIError(errorToString(topicActionsError), { api: "getTopicActions" });
+  if (vulnError) throw new APIError(errorToString(vulnError), { api: "getVuln" });
+  if (vulnActionsError)
+    throw new APIError(errorToString(vulnActionsError), { api: "getVulnActions" });
   if (pteamIsLoading) return <>Now loading PTeam...</>;
-  if (serviceTagIsLoading) return <>Now loading Tag...</>;
-  if (topicIsLoading) return <>Now loading Topic...</>;
-  if (topicActionsIsLoading) return <>Now loading TopicActions...</>;
+  if (vulnIsLoading) return <>Now loading Vuln...</>;
+  if (vulnActionsIsLoading) return <>Now loading VulnActions...</>;
 
   const service = pteam.services.find((service) => service.service_id === serviceId);
   if (!service) throw new Error("No such service.");
-  const matchedTopicTag =
-    topic.tags.find((topicTag) => topicTag.tag_id === serviceTag.tag_id) ||
-    topic.tags.find((topicTag) => topicTag.tag_id === serviceTag.parent_id);
-  if (!matchedTopicTag) throw new Error("Mismatch tagId for the topic.");
+
+  // vulnerable_package: package obtained by packageId
+  const vulnerable_package = vuln.vulnerable_packages.find((pkg) => pkg.package_id === packageId);
+  if (!vulnerable_package) throw new Error("No such vulnerable_package for this packageId.");
```

Comment:
```suggestion
  if (!vulnerable_package) throw new Error("No packageId for the vuln");
```
URLに指定されたpackageIdが間違っていることを伝えるべき